### PR TITLE
Create 19-12-Episode-55.md

### PR DESCRIPTION
**Pull request by yamalight**

### CODE REVIEW

**Comment by sw-yx** [comment]
hi! i'm a fan of the podcast, but i was wondering if you could include these links in the show notes of the podcast? so that i can access them while traveling. otherwise its hard to follow up on your articles!

thanks for your work!

**Comment by yamalight** [comment]
hey @sw-yx! unfortunately youtube has a character limit on description, so if I'd to include the links - only part of them would fit 😞 
I can only suggest to use [bxjs.dev](https://bxjs.dev) website for reference

**Comment by sw-yx** [comment]
ah. i was talking about the audio podcast since i mainly listen via audio and stop when something seems interesting

**Comment by sw-yx** [comment]
ah. i was talking about the audio podcast since i mainly listen via audio and stop when something seems interesting. but the website seems fine!
